inspirehep/base/templates/search/easy_search.html

```diff
@@ -0,0 +1,222 @@
+    <label for="inputCollaboration" class="col-sm-2 control-label">Collaboration</label>
+    <div class="col-sm-10">
+      <input style="width: 97%;" class="form-control" id="inputCollaboration" placeholder="Collaboration">
+    </div>
+  </div>
+  <div class="form-group">
+    <label for="inputKeywords" class="col-sm-2 control-label">Keywords</label>
+    <div class="col-sm-10">
+      <input  style="width: 97%;" class="form-control" id="inputKeywords" placeholder="Keywords">
+    </div>
+  </div>
+  <div class="form-group">
+    <label for="inputEprint" class="col-sm-2 control-label">Eprint</label>
+    <div class="col-sm-4 dropdown">
+      <button style="width:100%" class="btn btn-default dropdown-toggle" type="button" id="eprintDropdown" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
+        Any Type
+        <span class="caret"></span>
+      </button>
+      <ul class="dropdown-menu" id="eprintDropdownMenu" aria-labelledby="eprintDropdown">
+        <li><a>Any Type</a></li>
+        <li><a>ACC-PHYS</a></li>
+        <li><a>ASTRO-PH</a></li>
+        <li><a>GR-QC</a></li>
+        <li><a>HEP-EX</a></li>
+        <li><a>HEP-LAT</a></li>
+        <li><a>HEP-PH</a></li>
+        <li><a>HEP-TH</a></li>
+        <li><a>NUCL-EX</a></li>
+        <li><a>NUCL-TH</a></li>
+        <li><a>PHYSICS</a></li>
+        <li><a>QUANT-PH</a></li>
+      </ul>
+    </div>
+    <label for="inputNumber" class="col-sm-1 control-label">Number</label>
+    <div class="col-sm-5">
+      <input  style="width: 94%;" class="form-control" id="inputNumber" placeholder="Number">
+    </div>
+  </div>
+  <div class="form-group">
+    <label for="inputTopCite" class="col-sm-2 control-label">Topcite</label>
+    <div class="col-sm-4 dropdown">
+    <button style="width:100%;" class="btn btn-default dropdown-toggle" type="button" id="topciteDropdown" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
+        Don't care
+        <span class="caret"></span>
+      </button>
+      <ul class="dropdown-menu" id="topciteDropdownMenu" aria-labelledby="topciteDropdown">
+        <li><a>Don't care</a></li>
+        <li><a>50+</a></li>
+        <li><a>100+</a></li>
+        <li><a>250+</a></li>
+        <li><a>500+</a></li>
+        <li><a>1000+</a></li>
+        <li><a>50->99</a></li>
+        <li><a>100->249</a></li>
+        <li><a>250->499</a></li>
+        <li><a>500->999</a></li>
+      </ul>
+    </div>
+  </div>
+  <div class="form-group">
+    <label for="inputJournal" class="col-sm-2 control-label">Journal</label>
+    <div class="col-sm-4 dropdown">
+    <button style="width:100%;" class="btn btn-default dropdown-toggle" type="button" id="journalDropdown" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
+        Any Journal
+        <span class="caret"></span>
+      </button>
+      <ul class="dropdown-menu" id="journalDropdownmenu" aria-labelledby="topciteDropdown">
+        <li><a>Any Journal</a></li>
```

Comment:
How this list was generated?
In principle there are 3475 journal known to INSPIRE...